src/TypeScript/typescript.ts

```diff
@@ -0,0 +1,7 @@
+const myName = (person: string) => {
+  return "Hello, " + person;
+};
+//myName function that expects an arugument as a String
```

Comment:
I think you meant `argument` here and comments usually go above the thing they're explaining so I'd move this up.

It might also be nice to add a return type to the function signature (since it's returning a string) and also demonstrate the use of an interface.